dvc/commands/experiments/rename.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+import logging
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link
+from dvc.exceptions import InvalidArgumentError
+from dvc.ui import ui
+
+logger = logging.getLogger(__name__)
+
+
+class CmdExperimentsRename(CmdBase):
+    def check_arguments(self):
+        if not any(
+            [
+                self.args.rev,
+            ]
+        ) ^ bool(self.args.experiment):
+            raise InvalidArgumentError(
+                "Either provide an `experiment` argument, or use the `--rev` flag."
+            )
+
+    def run(self):
+        from dvc.utils import humanize
+
+        self.check_arguments()
+
+        renamed = self.repo.experiments.rename(
+            exp_name=self.args.experiment,
+            new_name=self.args.name,
+            rev=self.args.rev,
+            git_remote=self.args.git_remote,
+        )
+        if renamed:
+            ui.write(f"Renamed experiments: {humanize.join(map(repr, renamed))}")
+        else:
+            ui.write("No experiments to rename.")
+
+        return 0
+
+
+def add_parser(experiments_subparsers, parent_parser):
+    EXPERIMENTS_RENAME_HELP = "Rename experiments."
+    experiments_rename_parser = experiments_subparsers.add_parser(
+        "rename",
+        parents=[parent_parser],
+        description=append_doc_link(EXPERIMENTS_RENAME_HELP, "exp/rename"),
+        help=EXPERIMENTS_RENAME_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    rename_group = experiments_rename_parser.add_mutually_exclusive_group()
+    rename_group.add_argument(
+        "-g",
+        "--git-remote",
+        metavar="<git_remote>",
+        help="Name or URL of the Git remote to rename the experiment from",
+    )
+    default_msg = " (HEAD by default)"
+    msg = (
+        f"Rename experiments derived from the specified `<commit>` as "
+        f"baseline{default_msg}."
+    )
+    experiments_rename_parser.add_argument(
```

Comment:
I am not sure about the `rev` usage. It feels convenient but I would prefer to lie on the side of simplicity.

It would be also simple enough to pipe `dvc exp list` and this command to achieve the same and an example in the docs could be enough.

Reply:
Are you suggesting removal of the --rev flag all together? I agree, I believe it also came up in the docs review, that it is a little weird/unclear.

Reply:
Yes, I would prefer to just keep the simple positional arguments

Reply:
Sorry, I commented in the docs PR without looking at the core PR here closely. Agree with @daavoo, let's drop `--rev` please for now.

Reply:
Removed --rev flag